CMakeLists.txt

```diff
@@ -8,9 +8,11 @@ endif()
 
 if(WIN32)
   set(CMAKE_SHARED_LINKER_FLAGS "/FORCE:UNRESOLVED")
-  link_directories(${CMAKE_SOURCE_DIR}/thirdparty/pcre/win32/lib)
-  include_directories(${CMAKE_SOURCE_DIR}/thirdparty/getopt)
-  include_directories(${CMAKE_SOURCE_DIR}/thirdparty/pcre/win32/include)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+  set(BUILD_SHARED_LIBS TRUE)
+  link_directories(${CMAKE_SOURCE_DIR}/src/thirdparty/pcre/win32/lib)
```

Comment:
Alex - where do we stand with having pcre/getopt inside a 3rd party dir? Should we not move these to submodules if we can?

C

Reply:
I don't think these are in any repository anywhere. I grabbed both from an older version of SBF, and included them in our current tree. These dependencies will not change at all, so I think having them in our tree, as opposed to a submodule is easier.